### DISCUSSION TITLE
Do settings changes have any affect after the initial world generation?

### DISCUSSION THREAD

**Discussion by GibbousWaximus** [Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#3] [Q&A] [answered]
If I start playing with this mod installed, and later change the config to make larger dungeons, will that change have any affect? Or do all config changes need to occur before the world is first generated?

**Comment by Digitalroot** [comment, marked as answer]
The dungeons are created when you get close to them. This mod will improve any new dungeons you uncover. Same if you change the config value, the new value is used with any new dungeons.
